.github/workflows/pr.yml

```diff
@@ -40,3 +43,84 @@ jobs:
     with:
       python_version: ${{ needs.pyversion.outputs.pyversion }}
       poetry_version: 1.4.2
+
+  store_documentation:
+    name: Fetch the updated documentation
+    needs:
+      - code_format
+      - documentation_check
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+      - name: pack the documentation
+        working-directory: docs
+        run: tar --exclude .DS_Store --exclude sidebars.js -cvf documentation.tar *
+      - name: upload the documentation artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: documentation
+          path: docs/documentation.tar
+
+  pandev_pr:
+    name: Create a preview PR for pan.dev
+    needs: store_documentation
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: checkout pan.dev fork
+        uses: actions/checkout@v3
+        with:
+          repository: PaloAltoNetworks/pan.dev
+          token: ${{ secrets.CLSC_PAT }}
+
+      - name: download documentation artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: documentation
+          path: products/panos/docs
+
+      - name: unpack the documentation
+        working-directory: products/panos/docs
+        run: |
+          rm -rf 'panos-upgrade-assurance'
+          tar xvf documentation.tar
+          rm -f documentation.tar
+
+      - name: create a PR to upstream pan.dev
+        id: pr
+        uses: peter-evans/create-pull-request@v5
+        with:
+          push-to-fork: PaloAltoNetworks/panos-upgrade-assurance-pan.dev
+          token: ${{ secrets.CLSC_PAT }}
+          delete-branch: true
+          branch: "${{ github.event.pull_request.head.ref }}"
+          title: "[PAN-OS Upgrade Assurance][${{ github.event.pull_request.head.ref }}] documentation PREVIEW - do NOT MERGE"
```

Comment:
I also see that they tag the PRs with "DO NOT MERGE" tag on pan.dev repository PRs which are not meant to be merged. Can we also add that tag here?